webapp/sources/rudder/rudder-web/src/main/scala/bootstrap/liftweb/Boot.scala

```diff
@@ -709,20 +709,13 @@ class Boot extends Loggable {
     def utilitiesMenu = {
       // if we can't get the workflow property, default to false
       // (don't give rights if you don't know)
-      def workflowEnabled = RudderConfig.configService.rudder_workflow_enabled().either.runNow.getOrElse(false)
-      (Menu(MenuUtils.utilitiesMenu, <i class="fa fa-wrench"></i> ++ <span>Utilities</span>: NodeSeq) /
-      "secure" / "utilities" / "index" >>
-      TestAccess(() => {
-        if (
-          (workflowEnabled && (CurrentUser.checkRights(AuthorizationType.Validator.Read) || CurrentUser
-            .checkRights(AuthorizationType.Deployer.Read))) || CurrentUser
-            .checkRights(AuthorizationType.Administration.Read) || CurrentUser.checkRights(AuthorizationType.Technique.Read)
-        ) {
-          Empty
-        } else {
-          Full(RedirectWithState("/secure/index", redirection))
-        }
-      })).submenus(
+      (Menu(
+        MenuUtils.utilitiesMenu,
+        <i class="fa fa-wrench"></i> ++ <span>Utilities</span>: NodeSeq
+      ) /
+      "secure" / "utilities" / "index"
+      >> LocGroup("utilitiesGroup")
+      >> TestAccess(() => userIsAllowed("/secure/index", AuthorizationType.Administration.Read))).submenus(
```

Comment:
doesn't that will lock-out validator/etc who are not admin_read from accessing the change request validation screen ?

Reply:
I think it should only be the removal of Technique Read rights, i think it was there when techique editor was below Utilities, a looooong time ago